dbt/include/sqlserver/macros/adapters.sql

```diff
@@ -24,17 +24,17 @@
       case when table_type = 'BASE TABLE' then 'table'
            when table_type = 'VIEW' then 'view'
            else table_type
-      end as table_type
+      end as [type]
```

Comment:
any reason you changed this?

```suggestion
      end as table_type
```

Reply:
PostgreSQL adapter uses field name "type":
https://github.com/fishtown-analytics/dbt/blob/develop/plugins/postgres/dbt/include/postgres/macros/adapters.sql#L63